src/lite-yt-embed.js

```diff
@@ -35,6 +37,13 @@ class LiteYTEmbed extends HTMLElement {
           this.style.backgroundImage = `url("${this.posterUrl}")`;
         }
 
+        if( this.title && this.title !== 'null' ) {
+            const titleEl = document.createElement('h1');
```

Comment:
I'm not sure it's safe to use a `h1` for the title element as the page will most likely already have a `h1` and we'll break the document outline by duplicating it. Even inside a custom element, browsers will treat a `h1` as part of the top-level document's outline/heading structure.

With components/custom elements it's impossible to know what context they'll be used in so it might be better to provide an option for the user to specify what element could be used, perhaps defaulting to a `span` as that is neutral?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#multiple_h1 has a bit more info:

> Using more than one &lt;h1&gt; is allowed by the HTML specification, but is not considered a best practice. Using only one &lt;h1&gt; is beneficial for screenreader users.
>
> The HTML specification includes the concept of an outline formed by the use of &lt;section&gt; elements. If this were implemented it would enable the use of multiple &lt;h1&gt; elements, giving user agents—including screen readers—a way to understand that an &lt;h1&gt; nested inside a defined section is a subheading. This functionality has never been implemented; therefore it is important to use your headings to describe the outline of your document.

Reply:
Thanks @philwolstenholme . I'll add an attribute something like `title-tag=""`. Do you think that value needs to be sanitized? I'd rather not add the overhead of a regex like `replace(/[^a-z]/g,'')`, but if some one were to do something stupid like `<lite-youtube title="Hello" title-tag="<h2>"></lite-youtube>`, we'd get an uncaught error. 

I'm not sure how much error checking I should do.

Reply:
I'd be tempted to go for something like `title-element="h2"` — then you can pass the value straight into `createElement()`. It might be a bit more self explanatory too, and combined with an example in the documentation/README we might be able to get away without any validation/error checking.

Reply:
Thanks Phil. I've handled this in df45a2e9da4fc22bdec62ccd961a8af155f36f2b